modules/nw-infw-operator-installing.adoc

```diff
@@ -0,0 +1,152 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-node-firewall-operator.adoc
+
+:_content-type: PROCEDURE
+[id="installing-infw-operator_{context}"]
+= Installing Ingress Node Firewall Operator
+
+As a cluster administrator, you can install the Ingress Node Firewall Operator by using the {product-title} CLI or the web console.
+
+[id="install-operator-cli_{context}"]
+== CLI: Installing the Ingress Node Firewall Operator
+
+As a cluster administrator, you can install the Operator using the CLI.
+
+.Prerequisites
+
+* A cluster installed on bare-metal hardware with nodes that have hardware that support Ingress Node Firewall Operator.
```

Comment:
@msherif1234 Can you confirm this? As per me this is irrespective if hardware type provided the cluster has egress connectivity.

Reply:
@asood-rh can you confirm?

Reply:
@JoeAldinger I do not see any OCP cluster being called out as not supported. This is also dev preview feature so it was scoped testing for this release. Mohammad and Dan both are out of office. We should omit calling out bare metal. It is more of the operating system running on the cluster node and NIC driver's ability to filter packets. I agree with @anuragthehatter as we were able to test on vSphere without any problem.